pkg/apis/config/v1alpha4/default.go

```diff
@@ -54,11 +54,12 @@ func SetDefaultsCluster(obj *Cluster) {
 			obj.Networking.PodSubnet = "fd00:10:244::/64"
 		}
 	}
-	// default the service CIDR using the kubeadm default
+	// default the service CIDR using a different subnet than kubeadm default
 	// https://github.com/kubernetes/kubernetes/blob/746404f82a28e55e0b76ffa7e40306fb88eb3317/cmd/kubeadm/app/apis/kubeadm/v1beta2/defaults.go#L32
-	// Note: kubeadm is doing it already but this simplifies kind's logic
+	// Note: kubeadm is using a /12 subnet, that may allocate a 2^20 bitmap in etcd
+	// we allocate a /16 subnet that allows 65535 services (current Kubernetes tested limit is O(10k) services)
 	if obj.Networking.ServiceSubnet == "" {
-		obj.Networking.ServiceSubnet = "10.96.0.0/12"
+		obj.Networking.ServiceSubnet = "10.96.0.0/16"
```

Comment:
will this attempt to modify an existing cluster?

Reply:
Not currently. We have create / delete at the cluster level and then some ways to interact with the cluster, but no apply.

Reply:
`kind create cluster` is the only thing reading this at the moment.